apm-sniffer/apm-sdk-plugin/tomcat-7.x-8.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/tomcat78x/define/TomcatInstrumentation.java

```diff
@@ -45,6 +45,11 @@ public class TomcatInstrumentation extends ClassInstanceMethodsEnhancePluginDefi
      */
     private static final String EXCEPTION_INTERCEPT_CLASS = "org.apache.skywalking.apm.plugin.tomcat78x.TomcatExceptionInterceptor";
 
+    @Override
+    protected String[] witnessClasses() {
+        return new String[]{"javax.servlet.http.HttpServletResponse"};
```

Comment:
Does this only exist in Tomcat 10? This looks like a J2EE class. Why do you choose this as a witness class?